src/module-elasticsuite-core/Helper/IndexSettings.php

```diff
@@ -31,6 +37,21 @@ class IndexSettings extends AbstractConfiguration
      */
     const INDICES_SETTINGS_CONFIG_XML_PREFIX = 'indices_settings';
 
+    /**
+     * @var string
+     */
+    const INDICES_SETTINGS_CONFIG_XML_INDICES_PATTERN_PATH = 'smile_elasticsuite_core_base_settings/indices_settings/indices_pattern';
+
+    /**
+     * @var string
+     */
+    const DEFAULT_INDICES_PATTERN_NAME_OLD = '{{YYYYMMdd}}_{{HHmmss}}';
+
+    /**
+     * @var string
+     */
+    const DEFAULT_INDICES_PATTERN_NAME_NEW = '{{Ymd}}_{{His}}';
```

Comment:
Rename this one to "DEFAULT_INDICES_PATTERN"

Reply:
Done.